File: Leetcode 1295 Find Numbers with Even Number of Digits [easy] [array]/Find Numbers with Even Number of Digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findNumbers = function(nums) {
    let count = 0;
    for (let number of nums) {
        let num = number.toString();
        if (num.length % 2 === 0) {
            count++;
        }
    }
    
    return count;
};

File: Leetcode 1295 Find Numbers with Even Number of Digits [easy] [array]/Find Numbers with Even Number of Digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count = 0;
        for (int number : nums) {
            String num = String.valueOf(number);
            if (num.length() % 2 == 0) {
                count++;
            }
        }
        
        return count;
    }
}